packages/wrangler/src/dev.tsx

```diff
@@ -40,6 +40,7 @@ export type DevProps = {
   initialMode: "local" | "remote";
   jsxFactory: undefined | string;
   jsxFragment: undefined | string;
+  enablePersistence: boolean;
```

Comment:
BIKESHED: `enableLocalPersistence` or `localModePersistence` or something that signifies that this only affects local mode?